### ISSUE TITLE
Help wanted 

### ISSUE THREAD

**Issue by jmgiaever** [home-assistant-snap/home-assistant-snap#33]
Hi,

This project is at the moment only maintained by one person, and it's not possible for me to keep up with the releases and meet the users expectations. 

My seasonal job keeps me busy periodically and if this project requires changes to be able to build, it might slow down releases.

However I really like the idea of snap and would like this project to succeed, so if any of you would like to help out, reach out to me. 

This project also has so many implantations which I'm not able to test/debug, and to make it work properly in general, we need more maintainers. 

Thank you!

**Comment by neeks76** [comment]
Greeting,

Nice to meet you.
I'm one of person who interesting home assistant on snap.
Thanks for your support.

May I help help this project?
However, I'm worried, whatif I am interrupting to this project.

Because, I'm have not ability developer skills and mind.
Untill now, I'm working for computer networking.

In part of L2, L3, I can support this project.
And testing based on log.

If you don't mind it, I am willing to help this project.


God speed all in all

**Comment by hofst** [comment]
I have sent a donation your way and hope that you will be able to keep maintaining the project!

**Comment by jmgiaever** [comment]
Thank you for the donation. I'm donating the spare time I have, and it's highly appreciated.

**Comment by ThyMYthOS** [comment]
I would like to offer some help. I have some past experience with snapcraft maintaining the eclipse snap which is now handed over to snapcrafters. My production setup is currently running home-assistant-snap.

**Comment by jmgiaever** [comment]
Hi,

That would be highly appreciated - and helpful.

At the moment I want to port the snap over to core24, but it's been challenging as I haven't found a way to support installations of packages during runtime (HACS etc), which needs to be installed in a RW mount.

There seems to be issues also with the Python plugin for core24, but there are solutions available. I just haven't been able to get it working with the above requirement. 

And with the amount of work I currently have, I haven't had enough time to investigate it further.

**Comment by ThyMYthOS** [comment]
I've experienced similar issues porting my rather simple python based snaps to core24. In the short term I thInk we should establish a process to keep up with HA updates.

1. What quality gates does a new release have to fulfill?
2. What is the process for new releases?
3. Who can/is allowed to do which steps?

**Comment by jmgiaever** [comment]
Agreed. I wanted to be ahead since HA is already starting to complain about the SQLite version and the easiest to mind was to push further with a newer core. 

My process has been to release the final patch for each release cycle. E.g 2024.12.5. That means the snap package is always 1 minor release behind, as I release this when the minor version is released. 

At the moment it's just me that is maintaining this package, so just me having any access. If I understood your question correctly?

**Comment by ThyMYthOS** [comment]
OK got it. Maybe we can start by having an issue ticket for every planned release with a list of open points to complete until the release can be done. If it gets too convoluted in a single ticket we can use the project feature.

**Comment by jmgiaever** [comment]
Sounds great to me. I will have more time in about a week or so, but would you like to participate already now? I can invite you to this project.

**Comment by ThyMYthOS** [comment]
If you have a list of things to do for 2024.11 and 2024.12 I can start with that.

**Comment by ThyMYthOS** [comment]
@jmgiaever are you able to start a second home assistance instance using https://snapcraft.io/docs/parallel-installs? I get `execv failed: No such file or director` when I try to do so.
